sensor_array/analysis/array_analysis_SVD.py

```diff
@@ -35,9 +35,9 @@ def calculate_all_arrays(mof_list, array_size):
 
 def create_henrys_matrix(array, gases, kh_dataframe):
     df = kh_dataframe
-    H = [list(df.loc[df['MOF'].isin(array)][gas]) for gas in gases]
+    h_matrix = [list(df.loc[df['MOF'].isin(array)][gas]) for gas in gases]
 
-    return H
+    return h_matrix
```

Comment:
ah ok, this should have been squashed. What about that loc though?

Reply:
My bad on forgetting to squash. The ".loc" was unnecessary - deleted in next commit (Can squash if you'd prefer.)

Reply:
I'm going to merge this, but can you tell me why you are converting to a list here?